rankers/MinRanker/__init__.py

```diff
@@ -3,11 +3,17 @@
 
 class MinRanker(Chunk2DocRanker):
     """
-    :class:`MinRanker` calculates the score of the matched doc form the matched chunks. For each matched doc, the score
+    :class:`MinRanker` calculates the score of the matched doc from the matched chunks. For each matched doc, the score
```

Comment:
need to update the `manifest.yml` version for it to be `tested` and `pushed`. Same for MinRanker

Reply:
thanks, I updated the version for MinRanker and MaxRanker